modules/unzip/main.nf

```diff
@@ -0,0 +1,40 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process UNZIP {
+    tag '$bam'
+    label 'process_low'
```

Comment:
`process_low` reserves 2 cores I think. Do we actually have a label for processes that use only one CPU?

Reply:
[Nope](https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/conf/base.config),  never understood why tbh, I added [single-core versions for eager](https://github.com/nf-core/eager/blob/master/conf/base.config) but IIRC there wasn't interest adopting that.